tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -480,6 +480,102 @@ public function test_payment_fields_outputs_fields() {
+
+		remove_filter( 'woocommerce_is_checkout', '__return_true' );
+		update_option( WC_Stripe_Feature_Flags::CHECKOUT_SESSIONS_FEATURE_FLAG_NAME, $original_feature_flag );
+		WC_Stripe_Helper::update_main_stripe_settings( $original_stripe_settings );
+
+		$selector_div = '<div id="wc-stripe-currency-selector" class="wc-stripe-currency-selector"></div>';
+		if ( $expect_selector ) {
+			$this->assertStringContainsString( $selector_div, $output );
+		} else {
+			$this->assertStringNotContainsString( $selector_div, $output );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method test_payment_fields_renders_currency_selector_conditionally is not named in camelCase. (undefined)

(CamelCaseMethodName)